swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/Store.swift

```diff
@@ -98,15 +98,17 @@ public final class Store: ObservableObject {
   /// reason and alert the user if it was due to receiving a 401 from the
   /// portal.
   private func maybeShowSignedOutAlert() {
-    Task {
+    Task.detached { [weak self] in
+      guard let self else { return }
+
       do {
         if let savedValue = try await self.vpnConfigurationManager.consumeStopReason(),
            let rawValue = Int(savedValue),
            let reason = NEProviderStopReason(rawValue: rawValue),
            case .authenticationCanceled = reason
         {
 #if os(macOS)
-          self.sessionNotification.showSignedOutAlertmacOS()
+          await self.sessionNotification.showSignedOutAlertmacOS()
```

Comment:
Same here? How can this need an `await` now when the function definition doesn't change?